README.md

```diff
@@ -1,25 +1,4 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
- [ ] Kindly use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) and customize it to make your readme professional.